source/xrefdelta.tex

```diff
@@ -198,6 +198,8 @@
 
 \movedxref{iterator.container}{iterator.range}
 
+\movedxref{range.view.ref}{range.ref.view}
+
```

Comment:
The intro text for this index says "All clause and subclause labels from ISO C ++ 2017 (...) are present in this document, with the exceptions described below."

There was no subclause [range.view.ref] in C++17, which suggests it shouldn't be listed here. Or the intro text should more vaguely refer to "prior revisions of this document" instead of specifying C++17.

Reply:
Agreed, this change is wrong.

Fixed.